hack/tests/e2e-ansible.sh

```diff
@@ -1,215 +1,13 @@
-    kubectl create configmap deleteme
-    trap_add 'kubectl delete --ignore-not-found configmap deleteme' EXIT
-
-    header_text "delete custom resource (Memcached CR)"
-    kubectl delete -f ${OPERATORDIR}/deploy/crds/ansible.example.com_v1alpha1_memcached_cr.yaml --wait=true
-    header_text "if the finalizer did not delete the configmap..."
-    if kubectl get configmap deleteme 2> /dev/null;
-    then
-        error_text "FAIL: the finalizer did not delete the configmap"
-        operator_logs
-        exit 1
-    fi
-
-    header_text "The deployment should get garbage collected, so we expect to fail getting the deployment."
-    if ! timeout 60s bash -c -- "while kubectl get deployment ${memcached_deployment} 2> /dev/null; do sleep 1; done";
-    then
-        error_text "FAIL: memcached Deployment did not get garbage collected"
-        operator_logs
-        exit 1
-    fi
-
-    header_text "Ensure that no errors appear in the log"
-    if kubectl logs deployment/memcached-operator| grep -i error;
-    then
-        error_text "FAIL: the operator log includes errors"
-        operator_logs
-        exit 1
-    fi
-}
-
-header_text "Creating and building the operator"
-pushd "$TMPDIR"
-operator-sdk new memcached-operator \
-  --api-version=ansible.example.com/v1alpha1 \
-  --kind=Memcached \
-  --type=ansible
-cp "$ROOTDIR/test/ansible-memcached/tasks.yml" memcached-operator/roles/memcached/tasks/main.yml
-cp "$ROOTDIR/test/ansible-memcached/defaults.yml" memcached-operator/roles/memcached/defaults/main.yml
-cp -a "$ROOTDIR/test/ansible-memcached/memfin" memcached-operator/roles/
-cat "$ROOTDIR/test/ansible-memcached/watches-finalizer.yaml" >> memcached-operator/watches.yaml
-# Append Foo kind to watches to test watching multiple Kinds
-cat "$ROOTDIR/test/ansible-memcached/watches-foo-kind.yaml" >> memcached-operator/watches.yaml
-
-
-pushd memcached-operator
-
-header_text "Adding a second Kind to test watching multiple GVKs"
-operator-sdk add crd --kind=Foo --api-version=ansible.example.com/v1alpha1
-sed -i".bak" -E -e 's/(FROM quay.io\/operator-framework\/ansible-operator)(:.*)?/\1:dev/g' build/Dockerfile; rm -f build/Dockerfile.bak
-operator-sdk build "$DEST_IMAGE"
-# If using a kind cluster, load the image into all nodes.
-load_image_if_kind "$DEST_IMAGE"
-sed -i".bak" -E -e "s|REPLACE_IMAGE|$DEST_IMAGE|g" deploy/operator.yaml; rm -f deploy/operator.yaml.bak
-sed -i".bak" -E -e 's|Always|Never|g' deploy/operator.yaml; rm -f deploy/operator.yaml.bak
-# kind has an issue with certain image registries (ex. redhat's), so use a
-# different test pod image.
-METRICS_TEST_IMAGE="fedora:latest"
-docker pull "$METRICS_TEST_IMAGE"
-# If using a kind cluster, load the metrics test image into all nodes.
-load_image_if_kind "$METRICS_TEST_IMAGE"
-
-OPERATORDIR="$(pwd)"
-
-trap_add 'remove_operator' EXIT
-deploy_operator
-test_operator
-remove_operator
-
-popd
-popd
+# TODO: Include test script after ansible plugins PR is merged.
```

Comment:
We can remove the full file and target.

Reply:
but do not need to do it in this PR we can do in a follow-up. Let's save Travis cycles.